cranelift/codegen/src/machinst/abi.rs

```diff
@@ -763,8 +773,7 @@ impl SigData {
 
     /// Get the number of return values expected.
     pub fn num_rets(&self) -> usize {
-        let len = self.ret_indices.end - self.ret_indices.start;
-        usize::try_from(len).unwrap()
+        usize::try_from(self.rets_length).unwrap()
```

Comment:
Unless I've missed something, you can use `usize::from` here as well, right?

```suggestion
        usize::from(self.rets_length)
```